### PR TITLE
Add X-Forwarded-Port [Fixes #1220]

### PR DESCRIPTION
**Pull request by dtomcej**

### CODE REVIEW

File: forward/headers.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ const (
 	XForwardedFor      = "X-Forwarded-For"
 	XForwardedHost     = "X-Forwarded-Host"
 	XForwardedServer   = "X-Forwarded-Server"
+	XForwardedPort		= "X-Forwarded-Port"
```
Comment (timoreimann) [inline forward/headers.go:8]: Is the formatting off, or is it just Github?

Comment (dtomcej) [reply forward/headers.go:8]: WIP. not formatted yet.

**Comment by Starefossen** [comment]
Really looking forward to getting this into oxy and Træfik!

**Comment by ldez** [comment]
Close in favor of  #24

**Pull request closed without merging**